.gitmodules

```diff
@@ -7,3 +7,9 @@
 [submodule "lib/i18next"]
 	path = lib/i18next
 	url = https://github.com/i18next/i18next.git
+[submodule "lib/bootstrap"]
+	path = lib/bootstrap
+	url = https://github.com/twbs/bootstrap-sass/
+[submodule "lib/magnific-popup"]
+	path = lib/magnific-popup
+	url = https://github.com/dimsemenov/Magnific-Popup
```

Comment:
Why do you think this is not the right solution? 

I know that currently many people use bower and I also used it before, but in this scenario I see no advantage. Only one disadvantage: Bower does not create a git repository. If I have to change something in strophe.jingle or otr, with git submodule I can easily commit those changes.